doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/data-sets-authoring-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='data-sets-authoring-con_{context}']
+= Data sets authoring
+
+A data set is a collection of related sets of information. It can be stored in many ways, such as in a database, in a Microsoft Excel file, or in memory. A data set definition instructs {CENTRAL} methods to access, read, and parse a data set. {CENTRAL} does not store data. It enables you to define access to a data set regardless of where the data is stored.
```

Comment:
Minor rewrite suggestion.  Totally up to you.

A data set is a collection of related sets of information and can be stored in many ways. For example, in a database, in a Microsoft Excel file, or in memory. A data set definition instructs {CENTRAL} methods to access, read, and parse a data set. {CENTRAL} does not store data. It enables you to define access to a data set regardless of where the data is stored.

Reply:
Done. I have added your suggestions. :)